app/features/modelling/line-plot/index.js

```diff
@@ -2,27 +2,77 @@ import Chart from 'chart.js/auto';
 
 // Help at http://www.chartjs.org/docs/latest
 class LinePlotModel {
-  constructor(canvas, { frame, src }) {
+  constructor(canvas, { frame, src }, { onLoad, modelDidError }) {
     this.ctx = canvas.getContext('2d');
     console.log('Line Plot', this.ctx, frame, src);
```

Comment:
Unrelated to these changes, and I think actually helpful, just noting there are two console logs - here with constructor and line 29 after data loads.